### DISCUSSION TITLE
Windows Shell Context Menu

### DISCUSSION THREAD

**Discussion by s0nic9** [rejetto/hfs#347] [Q&A]
is it planned ?
🙏

**Comment by rejetto** [comment]
it's in my to-do, but not very high priority.
How often would you use it?

**Comment by s0nic9** [comment]
daily with hfs2.
Just switched to hfs3 only because embedded ssl is faster than stunnel over hfs2.

**Comment by rejetto** [comment]
on what port does your hfs3 has http? (not https, on localhost)

**Comment by s0nic9** [comment]
80

**Comment by rejetto** [comment]
i prepared this file.
execute it and it will add "Add to HFS (new)"
[windows-menu.reg.zip](https://github.com/rejetto/hfs/files/12645769/windows-menu.reg.zip)

let me know if it works well for you, it did for me.

**Reply by SanokKule** [reply]
If "Unprotected admin on localhost" is enabled it does indeed add a file/folder to the root of vfs.

**Reply by SanokKule** [reply]
Would be great if it could also generate and copy a link to the file.

**Reply by s0nic9** [reply]
nice ! thanks for fast solution. I'll test it during off-peaks hours.
meanwhile, I've found a sTunnel replacement (GoSecure) for HFS2 where speed isn't limited.

**Comment by rejetto** [comment]
it's an experimental solution, it's not "integrated" yet, and there's space for improvement.

**Comment by rejetto** [comment]
any feedback on this?

**Comment by rejetto** [comment]
https://github.com/rejetto/hfs/releases/tag/v0.49.0-alpha4

**Comment by greatphish** [comment]
still, expecting a new version with simple right-click context menu to add file/folder
btw: any way to hide the ps console window?

**Comment by rejetto** [comment]
> still, expecting a new version with simple right-click context menu to add file/folder

"expecting"?  the 0.49 beta is doing that

> btw: any way to hide the ps console window?

i'm not sure. I've found this parameter `-WindowStyle` that may work. I will experiment.
Suggestions are welcome

**Comment by rejetto** [comment]
final version 0.49 is published
